src/main/resources/sql/tables/industryInitialTablesSql.sql

```diff
@@ -6,6 +6,8 @@ CREATE INDEX IF NOT EXISTS SYS_IDX_FKN27UG21LW38SS31EXIAHHQH8B_10100 ON TBL_ITEM
 
 CREATE TABLE IF NOT EXISTS TBL_CART (ID BIGINT GENERATED BY DEFAULT AS IDENTITY (START WITH 1, INCREMENT BY 1) NOT NULL, USER_ID BIGINT, ITEM_ID BIGINT, NAME VARCHAR(255), DESCRIPTION VARCHAR(255), IMAGE VARCHAR(255), QUANTITY INTEGER, CONSTRAINT SYS_PK_10131 PRIMARY KEY (ID))
 
+CREATE TABLE IF NOT EXISTS TBL_ITEM_INVENTORY (ITEM_ID BIGINT NOT NULL, IN_STOCK INTEGER, CONSTRAINT SYS_PK_10098 PRIMARY KEY (ITEM_ID), CONSTRAINT TBL_ITEM_INVENTORY_FK FOREIGN KEY (ITEM_ID) REFERENCES TBL_ITEM(ID))
```

Comment:
Please add "GENERATED BY DEFAULT AS IDENTITY (START WITH 1, INCREMENT BY 1)" for ITEM_ID